core-bundle/assets/styles/component/clipboard.pcss

```diff
@@ -0,0 +1,18 @@
+a[data-controller="contao--clipboard"] {
+    &:before {
+        display: inline-block;
+        content: "";
+        width: 1em;
+        height: 1em;
+        margin-right: 2px;
+        margin-bottom: 2px;
+        vertical-align: middle;
+        background: url("../../../contao/themes/flexible/icons/clipboard.svg") no-repeat center center;
```

Comment:
Unfortunately, the icon you added says "remove from clipboard" or "close clipboard" instead of "copy to clipboard".

Reply:
We need a new one with a plus I guess.

Reply:
[This one](https://lucide.dev/icons/clipboard-plus), no?

Reply:
`clipboard-plus` looks too close to "close".
`clopboard-copy` seems more intuitive imo

Reply:
`clipboard-plus` is the only one of the three that says "add", so we should use this.

Reply:
Now using the `clipboard-plus` icon in 4a40e2cc601cc66672432e71e8027b92cd79a284.